### ISSUE TITLE
Create leaderboard

### ISSUE THREAD

**Issue by Edgeworth** [ZacharyForman/Gozzlang-Comp#1]
Would be nice to have some sort of leaderboard based on a representative (or full sample) of the judging data. You could either submit your own times (needing downloadable data) or each submission could provide a python script to build and run it (data remains secret).

**Comment by ZacharyForman** [comment]
I'd be interested in supporting a leaderboard - maybe we can run
submissions once a week (or more often, if they're generally easy to
automate running?) That would be on a best effort basis, though: I'm not
going to spend 3h getting a submission to run for a weekly test, but I will
for a final test round.

**Comment by Edgeworth** [comment]
Just require that submissions contain a python script that builds them, and produces an executable file called gozz in the current gozzrectory or something.

**Comment by ZacharyForman** [comment]
Yeah, that's fine, consider e.g. someone with a program that fails to run
on my machine.

**Comment by Edgeworth** [comment]
Just message them the error and do nothing.

**Comment by ZacharyForman** [comment]
Alright, I'll offer test runs once a week or so, starting on the 18th of February. This gives 18th, 25th, 4th and 11th as test runs.
